packages/gatsby/src/services/initialize.ts

```diff
@@ -295,6 +291,10 @@ export async function initialize({
     plugins: pluginsStr,
   })
 
+  // Start plugin runner which listens to the store
+  // and invokes Gatsby API based on actions.
+  startPluginRunner()
```

Comment:
wait until plugins are loaded so we can check which ones implement `onCreatePage`